Jenkinsfile

```diff
@@ -17,7 +17,7 @@ node('docker') {
     stage('Build') {
         dockerCleanup()
         docker.build(image.id, '--pull --no-cache .')
-        sh "docker tag ${image.id} docker-registry.usersys.redhat.com/${image.id}"
+        sh "docker tag ${image.id} registry.devshift.net/${image.id}"
```

Comment:
This won't work: https://github.com/fabric8-analytics/fabric8-analytics-worker/blob/master/Dockerfile.tests#L1

:)